README.md

```diff
@@ -153,6 +153,29 @@ JSX code blocks in the markdown file can be made interactive with the live attri
 
 Visit the documentation at [http://localhost:8000](http://localhost:8000) and navigate to see your README.md powered page and workbench. Changes to the README.md file will auto refesh the page.
 
+### Developing locally against MFE
+
+If you want to test the changes with local MFE setup, you need to create a "module.config.js" file in your MFE's directory containing local module overrides. After that the webpack build for your application will automatically pick your local version of Paragon and use it. The example of module.config.js file looks like this:
```

Comment:
I wonder if it might be helpful to also link to the frontend-build README that describes the module.config.js workflow in more depth?

https://github.com/edx/frontend-build#local-module-configuration-for-webpack

e.g.,

"For more details about module.config.js, refer to the frontend-build documentation."

Reply:
I was wondering about that too, I added the link, should be better now